src/Sarif/Comparers/ArtifactSortingComparer.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
I see several instances where using System.Linq is greyed out in VS, can you double check where this is actually needed and remove any unnecessary?

Reply:
It is required in this file, checked other files and removed the unnecessary using.